apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -1,7 +1,7 @@
 import { Theme } from '@fluentui-react-native/framework';
 import { FocusTrapZone, Separator, StealthButton } from '@fluentui/react-native';
 import { Button } from '@fluentui-react-native/experimental-button';
-import { Text } from '@fluentui-react-native/experimental-text';
+import { Text } from '@fluentui-react-native/text';
```

Comment:
Is this change only needed here? Don't you need to downgrade text in the other test pages too?

Reply:
I just tested the other pages and they seemed to be working for me. This change specifically targets the header on every page, so it does help with all pages.

Reply:
I'll take another look to see if this can be fixed in a different way than downgrading the Text for just the test header.

Reply:
Okay, sounds good. I see how downgrading the Text header works for now (it's the same solution I came to when working in the UWP tester recently) but I recall still seeing the issue pop up intermittently after that 😥 (particularly in the Tabs Experimental test page). That's why I wonder if we need to downgrade Text in all test pages.

Reply:
@ksiler if you're ok with this i'd like to check this change in because it makes the tester usable in UWP, and then tackle the intermittency and track it with #1046